docs/design/dp_datamodel.md

```diff
@@ -0,0 +1,58 @@
+
+Mizar's data-plane adopts a simple data model that is essential to extend its functionality. The data-model is independant of the API definitions of the management-plane. Typically, different management plane implementations that encorporated Mizar as its data-plane has a shim layer that translates management-plane data-model into Mizar's constructs. The model consists of endpoint, network, and VPC constructs.
```

Comment:
Typically, different management plane implementations that **incorporated** Mizar